data/faq/nl.yml

```diff
@@ -43,41 +46,80 @@
+  questions:
+    - question: Wat verandert er in 2017? 
+      answer: |
+        In grote lijnen:<br> 
+        - In 2017 willen we de eerste ronde via de computer doen verlopen. Het wedstrijdplatform daarvoor is volop in ontwikkeling.<br> 
+        - We trachten het inschrijvingsproces verder te automatiseren. <br>
+        - Om de groei te kunnen opvangen, zullen dit jaar tot 50 deelnemers in elke categorie tot de finale worden toegelaten.<br>
+        - De finale blijft dit jaar een proef op papier.<br>
+        - Er wordt een betere inschatting van vereiste tijdsduur en moeilijkheidsgraad nagestreefd voor de vragen.<br>
+        - Omdat computerklassen moeilijker te reserveren zijn, hopen we meer flexibiliteit in de praktische organisatie te kunnen toelaten.<br> 
+        Dit alles is afhankelijk van de uiteindelijke specificaties van het wedstrijdplatform, dat momenteel nog in ontwikkeling is. Tegen eind november zullen we concreter kunnen zijn.
     -
       question: Onder welke voorwaarden kan de eerste ronde in de school doorgaan ?
       answer: |
-        De eerste ronde kan in uw school worden georganiseerd indien aan al de volgende voorwaarden is voldaan:<br>
-        - Minstens 10 leerlingen van uw school moeten deelnemen, alle niveau's tesamen. <br>
-        - Er moet een lokaal beschikbaar zijn, geschikt voor een examen (voldoende plaats tussen de leerlingen, ...).<br>
+        De eerste ronde kan in uw school worden georganiseerd indien aan al de volgende voorwaarden is voldaan (noot: door de wijzingen in 2017 is het mogelijk dat dit later nog licht wordt aangepast):<br>
+        - Minstens 4 leerlingen van uw school moeten deelnemen, alle niveau's tesamen. <br>
+        - Er moet een computerlokaal beschikbaar zijn, met een internetverbinding en 1 computer voor iedere deelnemer.<br>
         - Er moet constant toezicht zijn tijdens de volledige proef.<br>
         - Het reglement van de proef (geen GSM, ...) moet gerespecteerd worden. Het wordt u enkele dagen voor de proef toegezonden. <br>
-        - De proef moet doorgaan op hetzelfde moment als in de andere centra. Dat wil zeggen, de proef moet beginnen op woensdag 27 januari 2016 tussen 13u en 14u30, en moet 1u30 duren. <em>(uitzondering voor scholen die de biologie-olympiade organiseren: zie hieronder)</em><br>
-        - U moet zelf alle vragenlijsten afdrukken op dinsdagmiddag of woensdagvoormiddag, en ze veilig bewaren tot het begin van de proef. <br>
-        - U moet ons de ingevulde opgaves (en zeker alle antwoordbladen) nadien doorsturen, ofwel door ze in te scannen en te emailen, ofwel via aangetekende post, ofwel door ze zelf af te geven in het verbetercentrum in Brussel (ULB Pleinlaan).
+        - De proef moet doorgaan op hetzelfde moment als in de andere centra. Voor editie 2017 betekent dit dat de proef plaatsvindt op woensdag 1 februari in de namiddag. De proef moet 1u30 duren. Er wordt later nog beslist hoeveel marge we daarop zullen toelaten.<br>
```

Comment:
discussed at the GA: the afternoon is probably too limiting (if they cannot put all students in the room which is probably the case). Letting them participate all the day make more sense.